meds_interp/meds_logistic_regression.py

```diff
@@ -13,21 +15,19 @@ def fit_logistic_regression(train_df: pl.DataFrame, test_df: pl.DataFrame, c) ->
     Returns:
         pl.Series: labels from logistic regression model for the test_df embeddings
     """
-    # embeddings = df.get_column("embeddings")
-    # labels = df.get_column("labels")
-
-    # classifier = LogisticRegression(C=c)
-    # classifier.fit(embeddings, labels)
-    # predicted_labels = classifier.predict(embeddings)
-    # return predicted_labels
-    embeddings = np.array(df['embeddings'].to_list())
-    labels = np.array(df['labels'])
-    
+
+    train_embeddings = np.array(train_df["embeddings"].to_list())
+    labels = np.array(train_df["label"])
+
+    test_embeddings = np.array(test_df["embeddings"].to_list())
+
     classifier = LogisticRegression(C=c)
-    classifier.fit(embeddings, labels)
-    predicted_labels = classifier.predict(embeddings)
+    classifier.fit(train_embeddings, labels)
+    predicted_labels = classifier.predict(test_embeddings)
+    predicted_probabilities = classifier.predict_proba(test_embeddings)[::, 1]
+
+    return pl.Series(predicted_labels), pl.Series(predicted_probabilities)
```

Comment:
just return the `pl.Series(predicted_probabilities)` as we can always retrieve labels using the probabilities